.github/workflows/introspection.yml

```diff
@@ -0,0 +1,32 @@
+name: Introspection CI
+
+on:
+  push:
+    branches:
+      - introspection
+  pull_request:
+    branches:
+      - introspection
+
+jobs:
+  ubuntu-latest-aurora-build-and-run-failover-tests:
```

Comment:
Do you just run a specific set of tests?

Reply:
Yes, just the unit tests most relevant to the introspection changes.

Reply:
Could it be shortened to "run-failover-tests-on-ubuntu-latest" ?
(There's no mention of aurora anywhere else in the yaml)

Reply:
This is to stay consistent with `main.yml`'s naming style

Reply:
On second thought, I will change the name. The purpose of this workflow is to ensure correctness while developing and debugging for introspection branch. This workflow will be removed once introspection is merged to the main branch.